.github/workflows/build-depends.yml

```diff
@@ -51,23 +51,38 @@ jobs:
             depends/sources
           key: depends-sources-${{ hashFiles('depends/packages/*') }}
           restore-keys: |
-            depends-sources-${{ hashFiles('depends/packages/*') }}
             depends-sources-
 
+      - name: Cache SDKs
+        uses: actions/cache@v4
+        if: inputs.build-target == 'mac'
+        with:
+          path: |
+            depends/sdk-sources
+            depends/SDKs
```

Comment:
why do we need both of these? if this is like depends, then one is compressed, and the other is usable. We should only cache the compressed one.

Reply:
Because if the extracted directory is present, it short-circuits the script ([source](https://github.com/dashpay/dash/blob/bb469687d3d936f82fd8e8fbe0934eec5e17df5e/contrib/containers/guix/scripts/setup-sdk#L18)).

Reply:
Dropped `depends/sdk-sources` in latest push